src/switch/_switch.scss

```diff
@@ -0,0 +1,68 @@
+.switch {
+  --switch-controller-bg: lightgreen;
+  --switch-active-slider-bg: white;
+  --switch-inactive-slider-bg: darkgrey;
+
+  position: relative;
+
+  display: inline-block;
+
+  width: 60px;
+  height: 34px;
+
+  &--is-disabled {
+    cursor: not-allowed;
+    pointer-events: none;
+  }
+}
+
+.switch__controller {
+  width: 0;
+  height: 0;
+
+  opacity: 0;
+
+  &:checked + .switch__slider {
+    background-color: var(--switch-controller-bg);
+  }
+
+  &:focus + .switch__slider {
+    box-shadow: 0 0 1px var(--switch-controller-bg);
+  }
+
+  &:checked + .switch__slider:before {
+    transform: translateX(26px);
+  }
+}
+
+.switch__slider {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  background-color: var(--switch-inactive-slider-bg);
+  border-radius: 34px;
+
+  cursor: pointer;
+
+  transition: 0.4s;
```

Comment:
Do we have a constant `--default-animation-duration` in this project? If not, it may be a good idea to define it and use it here (as far as i can see we also use animation in several places such as radio input and dropdown list item hover state)
This way all animations will be consistent, and package users can configure this value

Reply:
Yes, we already have. I didn't see that. I've changed with `$animation-duration`. Thank you 🎉